app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -594,5 +797,29 @@ public FeedItemListAdapter(MainActivity mainActivity) {
         protected void beforeBindViewHolder(EpisodeItemViewHolder holder, int pos) {
             holder.coverHolder.setVisibility(View.GONE);
         }
+
+        @Override
+        public void onCreateActionMode() {
+            mSpeedDialView.setVisibility(View.VISIBLE);
+        }
+
+        @Override
+        public void onDestroyActionMode_() {
+            mSpeedDialView.setVisibility(View.GONE);
+        }
+    }
+
+    private void close(@PluralsRes int msgId, int numItems) {
+        ((MainActivity) getActivity()).showSnackbarAbovePlayer(
+                getResources().getQuantityString(msgId, numItems, numItems), Snackbar.LENGTH_LONG);
+        getActivity().getSupportFragmentManager().popBackStack();
```

Comment:
What does this do on the episodes fragment? Is there anything that needs to be closed?

Reply:
@ByteHamster It was pulled from EpisodesApplyActionFragment which is used for multi select but will no longer be needed after this PR. Due to the context mode a fragment no longer needs closing. Renaming to 'showMessage'